### PR TITLE
add auth to docker execution

### PR DESCRIPTION
**Pull request by williamfalconeruk**
## Purpose
Add Docker auth credentials for circleci pipeline.

Fixes LPAL-170

## Approach

SEE Related PR on Make an LPA for full details: https://github.com/ministryofjustice/opg-lpa/pull/153

* [x] I have performed a self-review of my own code
* [x] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Purpose
 _Briefly describe the purpose of the change, and/or link to the JIRA ticket for context_
 
-Fixes LPA-####
+Fixes LPAL-####
```
Comment (charlesmmarshall) [inline docs/pull_request_template.md:4]: do we want this in a different PR?

**Review by charlesmmarshall — APPROVED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (williamfalconeruk) [reply docs/pull_request_template.md:4]: was literaly one character change spotted, so fixed whilst here.
